VideoWeb/VideoWeb/ClientApp/src/app/services/media-stream.service.ts

```diff
@@ -28,30 +28,42 @@ export class MediaStreamService {
     }
 
     getStreamForMic(device: UserMediaDevice): Observable<MediaStream> {
-        return from(this.navigator.mediaDevices.getUserMedia({ audio: { deviceId: { exact: device.deviceId } } })).pipe(
-            catchError(error => {
-                this.logger.error(`${this.loggerPrefix} Could not get audio stream for microphone`, error);
-                this.errorService.handlePexipError(new CallError(error.name), null);
-                return of(null);
-            })
-        );
+        return from(this.navigator.mediaDevices.getUserMedia({ audio: { deviceId: { exact: device.deviceId } } }))
+            .pipe(retry(3))
+            .pipe(
```

Comment:
These can be chained instead of separate pipe methods

Reply:
I can change this if you like. I tend to use multiple when the next pipe is going to be large e.g. line 46 onwards. As tracking down them commas becomes a pain.

Reply:
I prefer the multiple operators but it shouldn't make a difference and is just style guideline stuff. I'll resolve and you can go with what you have.